mono/utils/mono-os-wait-win32.c

```diff
@@ -152,6 +152,8 @@ mono_win32_wait_for_multiple_objects_ex (DWORD count, CONST HANDLE *handles, BOO
 	return result;
 }
 
+#if G_HAVE_API_SUPPORT(HAVE_CLASSIC_WINAPI_SUPPORT)
+
 DWORD
 mono_win32_signal_object_and_wait (HANDLE toSignal, HANDLE toWait, DWORD timeout, BOOL alertable)
```

Comment:
UWP does have SignalObjectAndWait if that is needed.

Reply:
As mentioned above, only in recent SDKs. We support older ones than that. We don't have to upstream this change if you are not comfortable with it.

Reply:
I don't like forking. Err toward change. Can improve it later if needed, which isn't likely.

Reply:
Ok, we'll imrpove this later if needed.

Reply:
Maybe add an implementation in mono-os-wait-win32-uwp.c similar to mono_win32_msg_wait_for_multiple_objects_ex for now?

Reply:
I'm a bit confused now. I see that `mono_win32_msg_wait_for_multiple_objects_ex` in this file (just below) is wrapped in the same `#if G_HAVE_API_SUPPORT(HAVE_CLASSIC_WINAPI_SUPPORT)` guard. Then it has a stub implementation for UWP in mono-os-wait-win32-uwp.c. Why don't we _need_ a similar stub implementation for `mono_win32_signal_object_and_wait` for UWP? Maybe it happens that nothing we need in the code on UWP actually calls `mono_win32_signal_object_and_wait`. @TautvydasZilys do you recall?

Reply:
Yeah we #ifdefed the callsite. It's an icall.

Reply:
Ok, thanks. I'd like to leave this as-is for now, since there is already a `#if G_HAVE_API_SUPPORT(HAVE_CLASSIC_WINAPI_SUPPORT)` for the function below. We can add an UWP stub later if it becomes necessary. @lateralusX, what do you think?

Reply:
OK, I can revisit this later.